docker/public/subdir/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Subdir Page</title>
```

Comment:
I suggest moving this index file to `docker/public/assets/index.html` and removing the `subdir`.
Also, change the title to `Assets Page` one is moved.